slick/src/sphinx/gettingstarted.rst

```diff
@@ -13,6 +13,9 @@ templates are created by the Slick team, with updated versions being made for ne
 
 * The `Slick Multi-DB Patterns template`_ shows you how to write Slick applications that can use
   different database systems and how to use custom database functions in Slick queries.
+  
+* The `Slick-starting-on-the-right-foot`_ shows you how write Slick applications using best practices. It also demonstrates unit testing and in general provides a seed for starting the product with Slick.
```

Comment:
I wouldn't call the cake pattern a best practice. It is one way to do things, but there are other ways with different trade-offs. Slick is not opinionated here. I would suggest to describe it more neutrally like "how to write a Slick application using the cake pattern".